dell/xps/13-9343/default.nix

```diff
@@ -0,0 +1,14 @@
+{ lib, pkgs, ... }:
+
+{
+  imports = [
+    ../../../common/cpu/intel
+    ../../../common/pc/laptop
+    ../../../common/pc/laptop/ssd
+  ];
+
+  services.throttled.enable = lib.mkDefault true;
```

Comment:
You can checkout this https://www.reddit.com/r/thinkpad/comments/870u0a/t480s_linux_throttling_bug/ and see if it applies to you.

Reply:
Oh I didn't realize `throttled` was to fix some problem. I probably don't need it.

But it's weird. If I test without it (with `s-tui with `stress`) my CPU cap at about 2420 Mhz. The clock doesn't seem to go down after like 10 secs like the person had in the linked issue.

If I test with `throttled`, my cpu cap at about 2350 Mhz.

It's also weird that my max with turbo is 2.70 GHz. I wonder if the temp doesn't allow it to go higher.

```
| Processor Base Frequency | 2.20 GHz |
| Max Turbo Frequency      | 2.70 GHz |
```
https://ark.intel.com/content/www/us/en/ark/products/85212/intel-core-i5-5200u-processor-3m-cache-up-to-2-70-ghz.html

Should I remove `throttled` and try to find out on my own why I don't hit 2.70 GHz?

Reply:
Yes. Just remove it in that case.

Reply:
Thanks